app/packages/state/src/recoil/atoms.ts

```diff
@@ -240,6 +240,11 @@ export const theme = atom<"dark" | "light">({
   ],
 });
 
+export const canEditSavedViews = atom({
+  key: "canEditSavedViews",
+  default: true,
```

Comment:
this will always be true in oss

Reply:
how would this get set in teams? Instead of defaulting to true regardless, can we check client type is oss, then default to true?